src/ReportManager/SchemaFormContext/SchemaForm/fields/Section/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import { SchemaFormContext } from '../../SchemaFormContext';
+import Header from '../Header';
+import Text from '../Text';
+import { FORM_FIELDS_TYPES } from '../../../../constants';
+import useFieldDetails from '../../useFieldDetails';
+
+const SECTION_ITEM_TYPES = {
+  HEADER: 'header',
+  FIELD: 'field'
+};
+
+const FORM_FIELDS = {
+  [FORM_FIELDS_TYPES.TEXT]: Text
+};
+
+const Section = ({ sectionName }) => {
+  const { getFieldUIType, isFieldActive } = useContext(SchemaFormContext);
+  const sectionDetails = useFieldDetails(sectionName);
+
+  const renderColumnSectionItems = (column) => column.map(({ name, type }) => {
+    if ( type === SECTION_ITEM_TYPES.FIELD && isFieldActive(name) ){
+      const Field = FORM_FIELDS[getFieldUIType(name)];
```

Comment:
If we already have a getter, why then we need to search in another object `FORM_FIELDS`? Why don't the getter just returns the field at once?